plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/BOAndDPCompileHook.java

```diff
@@ -25,7 +25,7 @@
     protected MavenRunner.Config mavenConfig;
     
     public BOAndDPCompileHook() {
-        System.out.println("Loaded BO and DP compile hook");
+        System.out.println("Loaded Blue Ocean and Declarative Pipelines compile hook");
```

Comment:
`Declarative Pipeline` is the [official name](https://jenkins.io/doc/book/pipeline/syntax/) 👍